build/pause/Dockerfile.windows

```diff
@@ -0,0 +1,24 @@
+# Copyright 2020 The Kubernetes Authors.
```

Comment:
For the sake of syntax highlighting in IDEs I think it makes sense to put this in `build/pause/windows/Dockerfile` and move linux to a similar location, `build/pause/linux/Dockerfile`. What do you think?

We may have to alter the docker context on `docker build`.